### PR TITLE
Typo codingrule

### PR DESCRIPTION
**Pull request by csjs0820**
apps/examples/ftpc/ftpc_cmds.c
106 line Exitting -> Exiting
apps/examples/kernel_sample/kernel_sample_main.c
491 line Exitting -> Exiting
apps/examples/sensorbd_demo/examples/i2c_tcs34725.c
181 line devie -> device
apps/examples/telnetd/telnetd.c
298 line tart -> start
apps/examples/fota_sample/fota_sample.c
103 line cancled -> cancelled

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [5ad52d370c30004d222a53ef30f8f7099d2253ca] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5ad52d370c30004d222a53ef30f8f7099d2253ca] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [0f4182a4aa9a4782f75e870594abc1800d124d23] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [0f4182a4aa9a4782f75e870594abc1800d124d23] - Code Rule Check (C++) OK.

File: apps/examples/fota_sample/fota_sample.c [hunk 1]
```diff
@@ -100,7 +100,7 @@ int fota_sample(int argc, char *argv[])
 	printf("Is OTA%d binary manually downloaded? If yes press Y to continue [Y/N]\n", next_part_id);
 	user_input = getchar();
 	if (user_input != 'Y' && user_input != 'y') {
-		printf(" fota update cancled !!!\n");
+		printf(" fota update canceled !!!\n");
```
Comment (thapav) [inline apps/examples/fota_sample/fota_sample.c]: cancelled*

Comment (csjs0820) [reply apps/examples/fota_sample/fota_sample.c]: @thapav Thank you!

**Comment by seinfra** [comment]
Target : [a8ad4fe4f0951e4cf5b6ec00f556a3895c25ba99] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a8ad4fe4f0951e4cf5b6ec00f556a3895c25ba99] - Code Rule Check (C++) OK.

**Pull request merged**